### PR TITLE
Get started contact form

### PR DESCRIPTION
**Pull request by JohnteFresh**
Add my contact form
Add my social contact links
Add Get my resume link

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -216,10 +331,114 @@ <h3 class="know-h3">Skills</h3>
...
+              <img src="icons/ic_twitter.png">
+            </a>
+
+          </li>
+          <li>
+
+            <a href="https://www.linkedin.com/in/johnson-irungu-85475a268/">
+              <img src="icons/ic_linkedin.png">
+            </a>
+
```
Comment (souad988) [inline index.html]: Great work was done in this milestone 👍🏼 , one small adjustement to do is the horizontal alignment of the get my resume button it needs to be centered,  and the vertical alignment of the social icons do your best to make this section identical to the template you are almost done 👍🏼 

**Your design**
Capture d’écran du 2023-07-18 12-54-47

**Figma design**
Capture d’écran du 2023-07-18 12-54-31

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @JohnteFresh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868


## Highlights
Good and Fair collaboration 
Use of email input type
Add validation to form inputs
Good use of form action attribute

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (JohnteFresh) [reply index.html]: Fixed

**Review by Gambit142 — APPROVED** [review]
Hi @JohnteFresh @melkamu372 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Use of semantic HTML ✔️ 
- Use of HTML validations ✔️ 
- Setup Formspree service correctly ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**